crates/pop-cli/src/main.rs

```diff
@@ -38,41 +40,126 @@ enum Commands {
 	Test(commands::test::TestArgs),
 }
 
+fn init_config() -> Result<()> {
```

Comment:
Minor, but would move this after main. One would expect to see the main function first for a binary, just so its quick to see what it is doing and to branch out from there.